recipes/genblasta/meta.yaml

```diff
@@ -0,0 +1,23 @@
+about:
+  home: http://genome.sfu.ca/genblast/download.html
+  license: 'GNU'
+  summary: 'genBlast is a program suite, consisting of two programs: genBlastA and genBlastG.
+      genBlastA parses local alignments, or high-scoring segment pairs (HSPs)
+      produced by local sequence alignment programs such as BLAST and WU-BLAST and identify groups of HSPs.'
+
+build:
+  number: 0
+  skip: True # [osx]
+
+package:
+  name: genblasta
+  version: '1.0.4'
+
+requirements:
+  run:
```

Comment:
Huh. I kind of feel like it might be too good to be true, but both builds with precompiled linux/osx test fine. What do you think?

This is the downloads page.

http://genome.sfu.ca/genblast/download.html